docs/tutorials/word2vec.ipynb

```diff
@@ -37,20 +37,20 @@
         "id": "AOpGoE2T-YXS"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.          for target in window[target_word]:
</pre><p>These words are really <code>context</code> rather than <code>target</code>. I think you can simplify that loop as</p><pre class="ql-syntax" spellcheck="false">for sample in negative_sampling_candidates:
  if sample in window[target_word]:
    break
</pre><p>If the window was a set, this lookup would be pretty efficient too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/text/pull/1229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>